src/clr-angular/popover/tooltip/tooltip-content.ts

```diff
@@ -19,25 +21,32 @@ const SIZES: string[] = ['xs', 'sm', 'md', 'lg'];
     `,
   host: {
     '[class.tooltip-content]': 'true',
-    // I'm giving up on animation, they did not work before and will not work now.
-    // Too many conflicts with Clarity UI.
     '[style.opacity]': '1',
+    role: 'tooltip',
```

Comment:
nit: do you mind changing it to`'[attr.role]': '"tooltip"'` as well?

Reply:
Yes. Will change.